.github/workflows/test_windows.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     runs-on: windows-latest
     strategy:
       matrix:
-        python-version: [ '3.6', '3.7', '3.8' ]
```

Comment:
@pedrocamargo no 3.9 on windows? Also, spacing is inconsistent

Reply:
Testing cannot be done because PyTables is not available for Python 3.9 yet (that's the HDF5 library used by OpenMatrix). In practice people can use by downloading the binaries from Chris Gohlke, but nothing we should put in here...